src/screens/DataApi/DataApi.tsx

```diff
@@ -90,4 +91,20 @@ const DataApi = () => {
   );
 };
 
+function getRegisterUrl(): string {
+  const search = window?.location?.search;
+  if (search !== undefined) {
+    const params = QueryString.parse(search);
+    const UTM_TAGS = ['utm_source', 'utm_medium', 'utm_campaign'];
+    const utmEntries = Object.entries(params).filter(entry =>
+      UTM_TAGS.includes(entry[0]),
+    );
+    if (utmEntries.length > 0) {
+      const qs = QueryString.stringify(Object.fromEntries(utmEntries));
+      return `https://apidocs.covidactnow.org/?${qs}#register`;
```

Comment:
Note that the old URL was `https://apidocs.covidactnow.org/access/` but that redirected to `https://apidocs.covidactnow.org/#register` so I used that instead.  If `/access/` was doing something interesting, let me know.